dart/generate_short_url/lib/utils/api.dart

```diff
@@ -0,0 +1,8 @@
+class ApiInfo {
+  static const String bitly_url = "https://api-ssl.bitly.com/v4/shorten";
+  static const String bitly_token = "[TOKEN]"; // https://app.bitly.com/settings/api/
+  static const String bitly_group_guid = "[GUID]"; // https://dev.bitly.com/api-reference/#getGroups
+
+  static const String tinyurl_url = "https://api.tinyurl.com/create";
+  static const String tinyurl_token = "[TOKEN]"; // https://tinyurl.com/app/settings/api
```

Comment:
Would you please ensure configurable values are read through `req.variables` rather than being hardcoded in the code? For example:

https://github.com/appwrite/functions-starter/blob/53948b924db3bab390e03c4d10970add1b0de8bb/dart-2.17/lib/main.dart#L37

Reply:
@stnguyen90 I need some help, I can't seem to be able to set environment variables. How do we set env variables so that they can be accessed using req.variables?

I tried putting them in `.env` file of appwrite, and restarted it. But no luck.

Reply:
They're not really environment variables. They're passed in as part of the request.

Reply:
Ah, do you mean the variables appended to the URL after a `?`?

For eg - `localhost:3000/?apiKey=somekey`

Reply:
@stnguyen90 I have updated the function to now take configurable values through `req.variables`. PTAL!